pkg/test/oci-utils/oci_layout_test.go

```diff
@@ -330,6 +330,20 @@ func TestBaseOciLayoutUtils(t *testing.T) {
 
 		isSigned = olu.CheckManifestSignature(repo, manifestList[0].Digest)
 		So(isSigned, ShouldBeTrue)
+
+		// checkCosignSignature -> true (withReferrers)
```

Comment:
Let's split this into 2 tests: one for notation and another for cosign in order for cosign tests to check the unsigned image at the begining of the test.